### DISCUSSION TITLE
protected $hidden not adhered

### DISCUSSION THREAD

**Discussion by webhoes** [clickbar/laravel-magellan#89] [Q&A]
I want a field not to be shown in the geojson output from Magellan.
I added an protected $hidden array on the model that also has the Magellan trait.
This field stil shows up in the geojson.
How to make sure a field is not added in the properties array of a geojson.

**Comment by saibotk** [comment]
Hey!

Thanks for your request, could you clarify this a little? In what geojson output?
Using our `toGeojsonFeatureCollection` helper macro?

If yes, just copy that and adjust the `f.*` inside the SQL statement.
